### PR TITLE
Port to python3

### PR DESCRIPTION
**Pull request by MaxSem**
Bug: T219472

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ This project allows the [Community Tech bot](https://en.wikipedia.org/wiki/User:
 ## Installation
 Virtualenv is recommended:
 
-    virtualenv venv
+    python3 -m venv venv
```
Comment (aezell) [inline README.md:25]: I think `venv` is duplicated here?

**Review by aezell — CHANGES_REQUESTED** [review]
Looks good with just the one change to the `venv` typo.

File: README.md (hunk 1 above)
Comment (MaxSem) [reply README.md:25]: The first one is module name, the second one is destination directory.

Comment (aezell) [reply README.md:25]: I had no idea. Thanks!

**Review by aezell — APPROVED** [review]

**Pull request merged**